jest.config.base.js

```diff
@@ -18,7 +20,9 @@ module.exports = {
     'json',
     'node',
   ],
-  setupFilesAfterEnv: ['console-fail-test/setup.js'],
+  setupFilesAfterEnv: [
+    join(__dirname, 'node_modules/console-fail-test/setup.js'),
```

Comment:
`<rootDir>/node_modules/console-fail-test/setup.js` should work.

Reply:
I think the issue with using `<rootDir>` is that it's dependent where the `jest.config.js` is.
If you `cd` into one of the packages, each package folder has its own `jest.config.js`, so that means that in a package `<rootDir>` will mean `<root>/packages/foo`, not just `<root>` - i.e. the path will not exist.

By using `_dirname` it means this path is always relative to this root config file.